privaterelay/middleware.py

```diff
@@ -51,24 +54,22 @@ def __call__(self, request):
         return response
 
 
-def _get_metric_view_name(request):
-    if request.resolver_match:
-        view = request.resolver_match.func
-        return f"{view.__module__}.{view.__name__}"
-    return "<unknown_view>"
+class ResponseMetrics:
 
+    re_dockerflow = re.compile(r"/__(version|heartbeat|lbheartbeat)__/?$")
 
-class ResponseMetrics:
-    def __init__(self, get_response):
+    def __init__(self, get_response: Callable[[HttpRequest], HttpResponse]) -> None:
         self.get_response = get_response
+        self.middleware = RelayStaticFilesMiddleware()
+
+    def __call__(self, request: HttpRequest) -> HttpResponse:
+        if not settings.STATSD_ENABLED:
+            return self.get_response(request)
```

Comment:
This is where the middleware is disabled at run-time, if `STATSD_ENABLED` is False.